src/store/metadata/context.json

```diff
@@ -140,6 +140,7 @@
     },
     "publishers": {
         "@id": "dcat:publisher",
+        "@type": "dcat:publisher",
```

Comment:
This is weird - I can see that it's needed, because it breaks when I remove it, but I feel like there should be a better solution (don't know what that solution is though!)